microsoft_intune/help.md

```diff
@@ -0,0 +1,99 @@
+# Description
+
+Integrate Microsoft Intune into your workflow
+
+# Key Features
+
+* Initiate a Windows Defender Antivirus scan on a machine.
+
+# Requirements
+
+* Azure credentials
+* Client ID, Client Secret and Tenant ID
+
+# Documentation
+## Setup
+
```

Comment:
Can you add quick instructions on how to obtain the necessary credentials to configure the connection. If there's a Microsoft guide that will work to. 

Here's an example from another plugin, https://github.com/rapid7/insightconnect-plugins/blob/master/cylance_protect/help.md#setup